src/reducers/url-state.js

```diff
@@ -199,7 +199,7 @@ const transforms: Reducer<TransformStacksPerThread> = (state = {}, action) => {
       // This may no longer be valid because of PII sanitization.
       const newTransforms = {};
       for (const [threadIndex, transformStack] of objectEntries(state)) {
-        const newThreadIndex = oldThreadIndexToNew.get(threadIndex);
+        const newThreadIndex = oldThreadIndexToNew.get(Number(threadIndex));
```

Comment:
wow

Reply:
Yeah, that was my first reaction as well 😄

Reply:
And sad thing is, our flow helper function doesn't work for that case. I tried to change the `objectEntries` signature to somehow convert the numbers to strings but couldn't make it work :/